packages/lib-utils/src/k8s/k8s-utils.ts

```diff
@@ -43,10 +65,16 @@ export const getK8sResourceURL = (
   const { ns, name, path, queryParams } = queryOptions;
   let resourcePath = getK8sAPIPath(model);
 
+  const activeWorkspace = getActiveWorkspace();
+  if (activeWorkspace) {
+    resourcePath += `/${activeWorkspace}`;
+  }
   if (resource?.metadata?.namespace) {
-    resourcePath += `/namespaces/${resource.metadata.namespace}`;
+    resourcePath += activeWorkspace
+      ? `/namespaces/${activeWorkspace}/${resource.metadata.namespace}`
```

Comment:
@BlakeHolifield I'm not sure this is the URL we expect based on our discussion with @rohitkrai03:
https://issues.redhat.com/browse/RHCLOUD-21699?focusedCommentId=21573215&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-21573215

`/workspaces/<workspace>/api/namespaces/<namespace>/applications`

I don't think we should inject `activeWorkspace` after `namespaces`